app/models/user/reputation_tokens/code_contribution_token.rb

```diff
@@ -1,18 +1,18 @@
 class User::ReputationTokens::CodeContributionToken < User::ReputationToken
-  params :repo, :pr_id
+  params :repo, :pr_id, :pr_number
```

Comment:
Adding a new parameter does mean that the existing reputation tokens will become invalid, but I think we can just drop them and then re-import all of them using the functionality (that will be introduced) in this PR.

Reply:
Yep. Correct.

In production, we'd bump the version of the i18n field here. Then the old ones wouldn't be invalid. But for now, we can just wipe and reimport.